hazelcast/src/main/java/com/hazelcast/map/impl/eviction/EvictionChecker.java

```diff
@@ -53,7 +53,7 @@
 public class EvictionChecker {
 
     protected static final double ONE_HUNDRED_PERCENT = 100D;
-    private static final int MIN_SANE_PER_PARTITION_SIZE = 2;
+    private static final int MIN_TRANSLATED_PARTITION_SIZE = 1;
```

Comment:
Minor: can it be just `MIN_PARTITION_SIZE`? I didn't like the "sane" attribute as it implicates that setting it too low is "insane" and I'm not sure what "translated" is referring to.